BinaryEncoding.md

```diff
@@ -63,17 +57,14 @@ sequence, then followed recursively by any child nodes.
 
 * Examples
   * Given a simple AST node: `I32Add(left: AstNode, right: AstNode)`
-    * First write the opcode for `I32Add` (uint8)
+    * First write the opcode for `I32Add` (`varuint32`)
```

Comment:
I'm not sure what the alternative is.  We'll surely hit the `uint8` limit of 256 opcodes (just from SIMD).  `uint16` would probably be "enough for anyone" (... probably) but will seriously bloat binaries.  LEB128 seems like the best thing here which takes 1 byte in the common case.  Remember, with opcode tables, ops can be reordered so that the hot ones which consume 99.9% of the bytes get the 1-byte encodings.